src/kaocha/plugin/cloverage.clj

```diff
@@ -119,11 +124,17 @@
                (contains? opts :cov-ns-exclude-regex)
                (update :ns-exclude-regex into (:cov-ns-exclude-regex opts))
 
+               (contains? opts :cov-test-ns-regex)
+               (update :test-ns-regex into (:cov-test-ns-regex opts))
+
+               :always
+               (update :test-ns-regex (partial mapv re-pattern))
+
                :always
-               (update :ns-regex (partial map re-pattern))
```

Comment:
mapv was maybe unnecessary, I was just confused since I got a test failure at first but it wasn't the () - [] the problem.